ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -1,157 +1,404 @@
 """Collection of tests for creation functions."""
 
 # global
+from enum import Flag
 import numpy as np
 import pytest
 from numbers import Number
 from hypothesis import given, strategies as st
 
 # local
 import ivy
+from ivy.functional.backends.numpy.data_type import dtype
 import ivy_tests.test_ivy.helpers as helpers
 import ivy.functional.backends.numpy as ivy_np
 
 
-# array
+# native_array - DONE
 @given(
-    dtype_and_x=helpers.dtype_and_values(ivy_np.valid_dtypes),
-    from_numpy=st.booleans(),
+    dtype_and_x=helpers.dtype_and_values(
+        ivy_np.valid_numeric_dtypes,
+        n_arrays=1,
+        min_value=None,
+        max_value=None,
+        allow_inf=False,
+        exclude_min=False,
+        exclude_max=False,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=10,
+        shape=None,
+        shared_dtype=True,),
```

Comment:
seems like you are just passing the default values for some of the arguments above. This is not necessary, just pass the arguments which are different from their default value.